snap/README.md

```diff
@@ -0,0 +1,149 @@
+# EdgeX Grove Device Service Snap
+[![snap store badge](https://raw.githubusercontent.com/snapcore/snap-store-badges/master/EN/%5BEN%5D-snap-store-black-uneditable.png)](https://snapcraft.io/edgex-device-grove)
+
+This folder contains snap packaging for the EdgeX Grove Device Service Snap
+
+The snap currently supports `arm64` platform.
+
+## Installation
+
+### Installing snapd
+The snap can be installed on any system that supports snaps. 
+You can see how to install snaps on your system [here](https://snapcraft.io/docs/installing-snapd/6735).
+
+However for full security confinement, the snap should be installed on an
+Ubuntu 18.04 LTS or later (Desktop or Server), or a system running Ubuntu Core 18 or later.
+
+### Installing EdgeX Device Grove as a snap
+The snap is published in the snap store as [edgex-device-grove](https://snapcraft.io/edgex-device-grove).
+You can see the current revisions available for your machine's architecture by running the command:
+```bash
+snap info edgex-device-grove
+```
+
+The latest beta version of the snap can be installed using:
+```bash
+
+sudo snap install edgex-device-grove --beta
+```
+
+The latest development version of the snap can be installed using:
+```bash
+sudo snap install edgex-device-grove --edge
+```
+
+## Snap configuration
+
+Device services implement a service dependency check on startup which ensures that 
+all of the runtime dependencies of a particular service are met before the service transitions to an active state.
```

Comment:
I think also true for v1 services?

Reply:
I am not sure about this one. Does this [build_depth.sh](https://github.com/edgexfoundry/device-grove-c/blob/main/scripts/build_deps.sh) count as dependency checks?

BTW, the service can not be started, and here are logs:
```
2021-12-16T17:26:21Z edgex-device-grove.device-grove[12956]: Error: 1: Unable to open configuration file
2021-12-16T17:26:21Z systemd[1]: snap.edgex-device-grove.device-grove.service: Main process exited, code=exited, status=1/FAILURE
2021-12-16T17:26:21Z systemd[1]: snap.edgex-device-grove.device-grove.service: Failed with result 'exit-code'.
...
2021-12-16T17:26:21Z systemd[1]: Failed to start Service for snap application edgex-device-grove.device-grove.
```

Reply:
Then let's remove it:
```suggestion

```

Reply:
Then let's remove it:
```suggestion

```
I'll comment on the error separately.